tests/test_pytype.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+import sys
+import pytest
+
+
+@pytest.mark.skipif(
+    sys.version_info >= (3, 13), reason="Need python < 3.13 for pytype"
+)
+def test_pytype():
+    result = subprocess.run(
+        ["pytype", "vulture/core.py", "--disable=attribute-error"],
```

Comment:
Instead of disabling the error, let's define `self.noqa_lines = {}` in the constructor alongside the other member variables.